File: sdk-java/src/main/java/com/crosscert/passkey/sdk/dto/AuthenticationFinishRequest.java
package com.crosscert.passkey.sdk.dto;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.databind.JsonNode;

public record AuthenticationFinishRequest(
    @JsonProperty("authenticationToken") String authenticationToken,
    @JsonProperty("publicKeyCredential") JsonNode publicKeyCredential
) {}

File: sdk-java/src/main/kotlin/com/crosscert/passkey/sdk/dto/AuthenticationFinishRequest.kt
package com.crosscert.passkey.sdk.dto

import com.fasterxml.jackson.annotation.JsonProperty
import com.fasterxml.jackson.databind.JsonNode

data class AuthenticationFinishRequest(
    @get:JvmName("authenticationToken") @get:JsonProperty("authenticationToken") @param:JsonProperty("authenticationToken") val authenticationToken: String,
    @get:JvmName("publicKeyCredential") @get:JsonProperty("publicKeyCredential") @param:JsonProperty("publicKeyCredential") val publicKeyCredential: JsonNode,
)
